keyboards/nckiibs/vaneelaex/keymaps/via/keymap.c

```diff
@@ -0,0 +1,103 @@
+ * |  +   |  1   |  2   |   3  |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO # |ISO / |  PGUP|  PGDN|      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+ * |  /   |  *   |  .   |   0  |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
+ * `---------------------------------------------------------------------------------------------------------------'
+ */
+[_RAISE] = LAYOUT_ss_6x12(
+  KC_KP_EQUAL,    KC_7,           KC_8,           KC_9,    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_DEL,
+  KC_KP_MINUS,    KC_4,           KC_5,           KC_6,    KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+  KC_KP_PLUS,     KC_1,           KC_2,           KC_3,    _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, _______,
+  KC_KP_SLASH,    KC_KP_ASTERISK, KC_KP_DOT,      KC_0,    _______, _______, _______, _______, _______, _______, _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
+),
+
+/* FN (Lower + Raise)
+ * ,---------------------------------------------------------------------------------------------------------------.
+ * | =    |   7  |   8  |   9  | Reset|      | Debug|      |      |      |      |      |      |      |      |  Del |
+ * |------+------+------+------+------+------+------+------+------+-------------+------+------+------+------+------|
+ * |  -   |  4   |   5  |   6  |      |      |      |Aud on|AudOff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
+ * |------+------+------+------+------+------+------+------+------+------|------+------+------+------+------+------|
+ * |  +   |  1   |  2   |   3  |      |Voice-|Voice+|Mus on|MusOff|MidiOn|MidOff|      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------+------+------+------+------|
+ * |  /   |  *   |  .   |   0  |      |      |      |      |      |             |      |      |      |      |      |
+ * `---------------------------------------------------------------------------------------------------------------'
+ */
+[_FN] = LAYOUT_ss_6x12(
+KC_KP_EQUAL,    KC_7,           KC_8,           KC_9,   RESET  , RGB_TOG, RGB_MOD, RGB_HUI, RGB_VAI, RGB_VAD, RGB_SAI, RGB_SAD, _______,TERM_ON, TERM_OFF, KC_DEL,
+KC_KP_MINUS,    KC_4,           KC_5,           KC_6,   _______, _______, MU_MOD,  AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, _______, _______, _______, _______, _______,
+KC_KP_PLUS,     KC_1,           KC_2,           KC_3,   _______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  _______, _______, _______, _______, _______,
+KC_KP_SLASH,    KC_KP_ASTERISK, KC_KP_DOT,      KC_0,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+)
+};
```

Comment:
```suggestion
};

layer_state_t layer_state_set_user(layer_state_t state) {
  state = update_tri_layer_state(state, _RAISE, _LOWER, _FN);
  return state;
}
```

Reply:
@drashna told me that VIA has some custom keycodes to switch layers, so I was asked to remove these lines in the last request. Can you clarify what I should follow, please?
https://github.com/qmk/qmk_firmware/pull/9817#discussion_r464779728